### PR TITLE
Add scheme files

### PR DESCRIPTION
**Pull request by MizukiSonoko**
Add protobuf's scheme.

## protobuf version 
3.3.0

## feature
- transaction header contains  a version number
https://github.com/hyperledger/iroha/compare/mvp-base...feature/mvp/add-scheme?expand=1#diff-20668a62f9784b9d6011379ec2f56285R10
- the package name is `iroha.protocol;`

### CODE REVIEW

File: schema/asset.proto [hunk 1]
```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+package iroha.protocol;
+
+// For Add/Subtract/Transfer
+message Asset{
+  string asset_id   = 1;
+  int64  amount     = 2;
```
Comment (takemiyamakoto) [inline schema/asset.proto]: should this be uint?

File: schema/asset.proto [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  int32  precision  = 3;
```
Comment (takemiyamakoto) [inline schema/asset.proto]: should this uint? Does protobuf support that?

File: schema/asset.proto (hunk 1 above)
Comment (Warchant) [inline schema/asset.proto]: there should be 
```
message uint256  {
  uint64 a;
  uint64 b;
  uint64 c;
  uint64 d;
}
```

File: schema/asset.proto (hunk 2 above)
Comment (Warchant) [inline schema/asset.proto]: precision should be uint8

**Comment by Warchant** [comment]
Ok I will merge, but I will fix schema in some places

**Pull request merged**